### PR TITLE
{bio}[intel/2020b] GTDB-Tk v1.5.0, HMMER v3.3.2, FastTree v2.1.11 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

- [x] #12795 (FastANI)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3534.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/53a0909d281d8e4b68dbfc5d4fcb4c12 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (3 easyconfigs in total)
node3101.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/f9b75b23f1bc850cd146e69fca14d977 for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/f/FastTree/FastTree-2.1.11-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+# HTTPS cert error:
+# hostname 'www.microbesonline.org' doesn't match either of 'genomics.lbl.gov', 'mojave.qb3.berkeley.edu', ...
+source_urls = ['http://www.microbesonline.org/fasttree/']
+sources = ['%(name)s-%(version)s.c']
+checksums = ['9026ae550307374be92913d3098f8d44187d30bea07902b9dcbfb123eaa2050f']
+
+builddependencies = [('binutils', '2.35')]
+
+skipsteps = ['source']
```
Comment (Micket) [inline easybuild/easyconfigs/f/FastTree/FastTree-2.1.11-GCCcore-10.2.0.eb:26]: Do this https://github.com/easybuilders/easybuild-easyconfigs/pull/12794#issuecomment-830578384 here as well?

Comment (boegel) [reply easybuild/easyconfigs/f/FastTree/FastTree-2.1.11-GCCcore-10.2.0.eb]: done in b9ab9ccb35

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS="FastTree-2.1.11-GCCcore-10.2.0.eb"

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS="FastTree-2.1.11-GCCcore-10.2.0.eb"

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS="FastTree-2.1.11-GCCcore-10.2.0.eb"

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 7 out of 7 (3 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/eb136a73d110fabd37ea7cfcdaf2c488 for a full test report.

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**